src/routes/Sidebar.svelte

```diff
@@ -92,13 +81,18 @@
 								Create a new project
 							</Command.Item>
 						</AlertDialog.Trigger>
```

Comment:
If I click in the project name input, the dialog closes. I think the reason is that focus of the Popover is lost which means it is closed, unmounting the AlertDialog component.